core/src/main/scala/org/apache/spark/executor/CoarseGrainedExecutorBackend.scala

```diff
@@ -113,9 +113,9 @@ private[spark] class CoarseGrainedExecutorBackend(
 
     case Shutdown =>
       stopping.set(true)
-      executor.stop()
-      stop()
-      rpcEnv.shutdown()
```

Comment:
`rpcEnv.shutdown()` will be called in `executor.stop()` and `endpoint.stop()` will be called by `rpcEnv.shutdown()`, so we don't need these two lines.